src/helix-ui/styles/components/tabs.less

```diff
@@ -34,37 +27,32 @@ hx-tab {
 }
 
 hx-tabcontent {
-  background-color: inherit;
+  -ms-grid-columns: 1fr;
```

Comment:
please double check if this is necessary

Reply:
definitely needed, otherwise IE doesn't know how many columns are present in order to apply its arrangement algorithm